client/my-sites/backup/backup-date-picker/index.tsx

```diff
@@ -48,7 +49,13 @@ const BackupDatePicker: FC< Props > = ( { selectedDate, onDateChange } ) => {
 		? today.clone().subtract( visibleDays, 'days' )
 		: oldestDateAvailable;
 
-	const canGoToDate = useCanGoToDate( siteId, selectedDate, oldestDateAvailable );
+	const hasNoBackups = useSelector( ( state ) => {
+		const backups = getRewindBackups( state, siteId ) || [];
+		// in-progress backups should not be counted as backups, yet.
+		return ! backups.filter( ( backup ) => backup.status !== 'started' ).length;
```

Comment:
I think it's better to check if the backup is an array before we do a filter.

Reply:
@yashwin If I'm understanding you correctly, isn't the array type guaranteed on line 53 with `|| []`? I guess in the strictest sense, `getRewindBackups` could return a non-array, but that would be a violation of the JSDoc declaration at https://github.com/Automattic/wp-calypso/blob/7655b6d1f326371990851da10ebb97f133fbdc17/client/state/selectors/get-rewind-backups.js#L8

Reply:
Yes, makes sense 👍